pkg/db/migrations/3_topology.sql

```diff
@@ -2,55 +2,77 @@
 -- +goose StatementBegin
 
 
-CREATE TABLE system (
+
+CREATE TABLE IF NOT EXISTS SYSTEM_TEMPLATES (
```

Comment:
s/SYSTEM_TEMPLATES/system_templates/

This script should never run when the table already exists so the IF NOT EXISTS can come out. 
we also should not be making changes to this file, but rather adding additional migration scripts that alert the tables originally created here

Reply:
I found the If not exists is beneficial for the local development envs... considering the case when I just need to alter one of the migration script; delete the goose_db_version and the target table... If I don't have the if not exists I would either need to do delete all the tables or use a different DB altogether.